### ISSUE TITLE
gptfdisk: SIGSEGV (Address boundary error) (sgdisk --version)

### ISSUE THREAD

**Issue by unitythemaker** [NixOS/nixpkgs#224747]
### Describe the bug
When I try to create a partition no matter what format it is, I get the following error:
```
Error creating partition on /dev/sda: Failed to determine /nix/store/c617hvfbz0fmsg17h8ss13gcxl6bqlba-gptfdisk-1.0.9/bin/sgdisk's version from: Process killed with a signal (udisks-error-quark, 0)
```

I tried to run it myself and got the error too:
```
unitythemaker@computhemaker ~> /nix/store/c617hvfbz0fmsg17h8ss13gcxl6bqlba-gptfdisk-1.0.9/bin/sgdisk --version
GPT fdisk (sgdisk) version 1.0.9

fish: Job 1, '/nix/store/c617hvfbz0fmsg17h8ss…' terminated by signal SIGSEGV (Address boundary error)
unitythemaker@computhemaker ~ [SIGSEGV]>
```

### Steps To Reproduce
1. Install ``gnome.gnome-disk-utility`` if not installed already.
2. Run the app named Disks (or cmdline: ``gnome-disks``)
3. Try to create a partition (options does not matter)

### Expected behavior
I was expecting the Disks app to create the partition successfully.

### Screenshots
![Screenshot from 2023-04-05 03-50-13](https://user-images.githubusercontent.com/83316880/229954135-9381f0a1-bc75-4795-894c-27e0a1e6fef5.png)

### Additional context
This issue might be the duplicate of #62749 which is closed but still happening. This issue is probably not related to ``gnome-disks`` but to ``gptfdisk``.

### Notify maintainers
@jtojnar @ehmry 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
unitythemaker@computhemaker ~> nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.105, NixOS, 22.11 (Raccoon), 22.11.3459.884e3b68be0`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.1`
 - channels(root): `"home-manager, nixos-22.11"`
 - channels(unitythemaker): `"home-manager-22.11.tar.gz"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by unitythemaker** [comment]
Found the exact duplicate, closing the issue: #219775 
Possible fix for it: #221066

**Comment by ElnuDev** [comment]
Seems to no longer be an issue after updating to 6c43a3495a11e261e5f41e5d7eda2d71dae1b2fe.
